.teamcity/_self/projects/WPComTests.kt

```diff
@@ -207,6 +207,178 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
 	}
 }
 
+fun gutenbergPlaywrightBuildType( viewportName: String, buildUuid: String ): BuildType {
```

Comment:
For this function I copied over existing function `gutenbergBuildType` and made some changes.